xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -1760,13 +1763,31 @@ glusterd_volinfo_find_by_volume_id(uuid_t volume_id,
     return ret;
 }
 
+static int
+match(struct cds_lfht_node *tmp_node, const void *_key)
+{
+    glusterd_volinfo_node *node = caa_container_of(tmp_node,
+                                                   glusterd_volinfo_node, next);
+    char key[NAME_MAX + 1] = "";
```

Comment:
nit: is that needed? I see in gf_strncpy() that it's being done:
```
dest[dest_size - 1] = '\0';
```

Also, if we already do a strlen(key), why not use it and copy as much as we need?

Reply:
For the gf_strncpy(),
I will make the update here.
But in many places, it is declared in this way.

Also, will update to use strlen(key) instead of sizeof(key).